### ISSUE TITLE
Use with Audited Gem

### ISSUE THREAD

**Issue by ACPK** [thoughtbot/administrate#616]
When I am using https://github.com/collectiveidea/audited, I get:

```
uninitialized constant Audited::AuditDashboard
```

**Comment by ACPK** [comment]
Closing it as the bug is due to https://github.com/thoughtbot/administrate/issues/179.

**Comment by turgs** [comment]
This is still an issue today... even after #179 has been merged in

**Comment by ACPK** [comment]
@turgs - I reopened the issue.

**Comment by pablobm** [comment]
Thank you for filling this issue. Would you be able to provide more information?

* The full exception backtrace.
* Do you have admin routes to Audited models?
* Are Audited models mentioned on any dashboard?

**Comment by nickcharlton** [comment]
I'm going to close this as it's been a long time with open questions; if it's a namespacing issue (which is common with these sorts of gems), it'd be better addressed in a new issue.
